kube-monitoring/charts/templates/_datasource.tpl

```diff
@@ -0,0 +1,70 @@
+# Configuration file version
+apiVersion: 1
+
+# List of data sources to delete from the database.
+deleteDatasources:
```

Comment:
is this necessary if the list is empty?

Reply:
I intend to leave each field here, even if it is not used, to facilitate additional configurations